src/splitInteger.test.js

```diff
@@ -3,19 +3,27 @@
 const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
+  if a value is divisible by a numberOfParts`, function() {
+  const goals = splitInteger(6, 2);
```

Comment:
We can remove goals const and move splitInteger(6, 2) inside expect(). It will decrease code length. Please check everywhere